src/components/PrivacyCheckbox.stories.mdx

```diff
@@ -0,0 +1,66 @@
+import {ArgsTable, Canvas, Meta, Story} from '@storybook/addon-docs';
+import {userEvent, within} from '@storybook/testing-library';
+import {expect} from '@storybook/jest';
+
+import PrivacyCheckbox from "./PrivacyCheckbox";
```

Comment:
```suggestion
import PrivacyCheckbox from './PrivacyCheckbox';
```
... we should add `.mdx` to the prettier extensions in the `package.json` scripts for `format` and `checkformat`